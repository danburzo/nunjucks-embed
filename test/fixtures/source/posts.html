<h1>Posts: {{ posts.length }}</h1>
{% for post in posts %}
	{% embed "components/post.html" %}
		{% block title %}{{ post.title }}{% endblock %}
		{% block excerpt %}{{ post.excerpt }}{% endblock %}
	{% endembed %}
{% endfor %}