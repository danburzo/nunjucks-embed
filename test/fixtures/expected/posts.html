<h1>Posts: 3</h1>
<article>
	<h1>Post 1</h1>
	<p>Excerpt 1</p>
</article>
<article>
	<h1>Post 2</h1>
	<p>Excerpt 2</p>
</article>
<article>
	<h1>Post 3</h1>
	<p>Excerpt 3</p>
</article>